README.md

```diff
@@ -1 +1,70 @@
+### Goal : Add Linters to html & css 📚
+<a name="readme-top"></a>
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)<br/>
+  This project is about the usage and benefits of <strong>ESLint</strong>.<br/>
+  ESLint is a static code analysis tool for finding errors, bugs, and stylistic errors.<br/>
+  This provides the developer with the ease of delivering clean code.<br/>
+  
+  - [🛠 Built With](#built-with)
+    - [HTML](#tech-stack)
+    - [CSS](#tech-stack)<br/>
+    
+    - [Key Features](#key-features):
+    - The most important part is to add the specified folders and files for the configuration of ESLINT.<br/>
+      Files like <strong>.gitignore </strong> or folders like <strong>.github/workflows</strong> are important to store<br/>
+      important files like <strong>linters.yml</strong>, as well as <strong>files to be hidden </strong>like node_modules are placed<br/>
+      inside the .gitignore file.<br/>
+      Another important part is the creation of branches. Besides our <strong><em>main</em></strong> branch, I created a branch <br/>
+      to develop the header which will be merged to the main branch.<br/>
+      
+  - [🚀 Live Demo](#live-demo)<br/>
+  
+     This is the live demo to <em>helloMicroverse project</em>: <br/>
+     https://ivonnebenitesrodriguez.github.io/helloMicroverse/
+    
+- [💻 Getting Started](#getting-started)
+  - 📍[Prerequisites](#prerequisites)<br/>
+  
+    - Install a code editor
+    - Install node.js 
+    
+  - 📍[Setup](#setup)
+  - Create a new repository to store files named index.html and style.css 
+  - Inside this repository, add a folder named .github/workflows
+  - Inside the folder named .github/workflows add the file named linters.yml
+  - Create a file named .gitignore and add node_modules to this file.
+  - Create a file named .hintrc to set up Webhint tool.
+  - Create a file named .stylelintrc.json to set up Stylelint tool.
+    
+  - 📍[Install](#install)
+  - Before installing node.js verify if you already have node <br/>
+    For it use this command:<strong> ```  node   -v ``` </strong> 
+  
+  - 📍[Usage](#usage)
+  - This tool helps developers to increase productivity and efficency.
+    
+- [ 🌸 Author](#authors) :  👩🏽‍💻 Ivonne Benites
+  - GitHub: [@githubhandle](https://github.com/IvonneBenitesRodriguez)
+  - Twitter: [@twitterhandle](https://twitter.com/IvonneBenitesR)
+  - LinkedIn: [LinkedIn](https://www.linkedin.com/in/ivonnebenites/)
+
+- [🔭 Future Features](#future-features)
+  - I will add the a navbar with 3 categories: Home, Products and Contact.
+  - I will add a section with a class. There I will add pictures and text.
+     
+- [🤝 Contributing](#contributing)
+  - Contributions, issues, and feature requests are welcome!
+    
+- [⭐️ Show your support](#support)
+  - If you like this project, a star will be very appreciated! ⭐️
+    
+- [📝 License](#license)
+  - This project is [MIT](./LICENSE) licensed.
+
```

Comment:
Hello 👋🏾 @tresorsawasawa 
I thank you for your review, 
I already made all the changes you suggested, and updated my README.md file according to my project information but following the syntax of the Microverse README template, can you please verify it  🙏🏿 . 
Since , sadly the system does not allow me to re-submit the project.
Thank you,